MANIFEST.in

```diff
@@ -32,7 +32,11 @@ recursive-include resolwe/test/tests/files *
 recursive-include resolwe/test/tests/processes *.yml
 recursive-include resolwe/toolkit/tests *.py
 recursive-include resolwe/toolkit/tests/files *
+recursive-include resolwe/storage/tests *.py
+include resolwe/storage/fixtures/*.yaml
 # include Dockerfiles and files needed to build Docker images
 recursive-include resolwe/toolkit/docker_images Dockerfile Dockerfile.* README.md
 include resolwe/toolkit/docker_images/base/curlprogress.py
 include resolwe/toolkit/docker_images/base/re-import.sh
+# Include encrypted credentials to test connectors.
+include tests/testing_credentials_*.json.enc
```

Comment:
This are Travis specific files, so it would be better to not include them like we don't include `.travis.yml`.